dev-python/openpyxl/openpyxl-2.6.2.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+
+inherit distutils-r1 vcs-snapshot
+
+DESCRIPTION="Pure python reader and writer of Excel OpenXML files"
+HOMEPAGE="https://openpyxl.readthedocs.io/en/stable/"
+SRC_URI="https://bitbucket.org/${PN}/${PN}/get/${PV}.tar.bz2 -> ${P}.tar.bz2"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+RDEPEND="
+	dev-python/jdcal[${PYTHON_USEDEP}]
+	dev-python/et_xmlfile[${PYTHON_USEDEP}]
+"
+DEPEND="
+	test? (
+		${RDEPEND}
+		dev-python/pytest[${PYTHON_USEDEP}]
+		dev-python/lxml[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP},tiff,jpeg]
+	)"
+BDEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+python_test() {
+	pytest -v || die "Testing failed with ${EPYTHON}"
```

Comment:
`-vv` is preferred to `-v`. I'm not sure that is properly documented on the wiki though :)

Reply:
Done.